mappings/net/minecraft/data/SnbtProvider.mapping

```diff
@@ -19,10 +28,12 @@ CLASS net/minecraft/class_2463 net/minecraft/data/SnbtProvider
 		FIELD field_20538 name Ljava/lang/String;
 		FIELD field_20539 bytes [B
 		FIELD field_20540 sha1 Ljava/lang/String;
+		FIELD field_24616 jsonizedContent Ljava/lang/String;
 		METHOD <init> (Ljava/lang/String;[BLjava/lang/String;Ljava/lang/String;)V
 			ARG 1 name
 			ARG 2 bytes
-			ARG 3 sha1
+			ARG 3 jsonizedContent
+			ARG 4 sha1
```

Comment:
Field declaration and constructor parameter are swapped in the code! I did go back and check this multiple time. You can thank Mahjong for this mess up.

Reply:
This is not really a bug. This should have been caught by matcher at the very least.

Reply:
I know, but just wanted to comment it in case someone else came in and had questions.